runtime/rasdump/javadump.cpp

```diff
@@ -1052,14 +1052,31 @@ JavaCoreDumpWriter::writeEnvironmentSection(void)
 	/* Write the running mode data */
 	_OutputStream.writeCharacters("1CIRUNNINGAS   Running as ");
 
+	/* If JITServer enabled, print running mode as JITServer and client UID*/
+#ifdef JITSERVER_SUPPORT
+	if (_VirtualMachine->internalVMFunctions->isJITServerEnabled(_VirtualMachine))
```

Comment:
Note that the JVM could be built with JITSERVER_SUPPORT, but still used in regular JVM mode. Your code does not allow for that andf it will print "Running as  JVM"